<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Меню и продукты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Блюда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название блюда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выход-вес </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>149</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>189</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продукты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название продукта</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>481</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>260</y>
     <width>291</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Состав блюда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Блюдо</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продукт</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Количество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Данные о блюдах</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Состав блюда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продукты</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>810</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>840</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>600</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Кг</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>литр</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>шт</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Единицы измерения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>680</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cостав блюда:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Посмотреть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>640</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
